packages/concerto-util/types/lib/errorcodes.d.ts

```diff
@@ -1,4 +1,46 @@
-export const DEFAULT_BASE_EXCEPTION: "DefaultBaseException";
-export const DEFAULT_VALIDATOR_EXCEPTION: "DefaultValidatorException";
-export const REGEX_VALIDATOR_EXCEPTION: "RegexValidatorException";
-export const TYPE_NOT_FOUND_EXCEPTION: "TypeNotFoundException";
+export namespace DEFAULT_BASE_EXCEPTION {
+    const status: string;
+    const code: string;
+}
+export namespace DEFAULT_VALIDATOR_EXCEPTION {
+    const status_1: string;
+    export { status_1 as status };
+    const code_1: string;
+    export { code_1 as code };
+}
+export namespace REGEX_VALIDATOR_EXCEPTION {
+    const status_2: string;
+    export { status_2 as status };
+    const code_2: string;
+    export { code_2 as code };
+}
+export namespace TYPE_NOT_FOUND_EXCEPTION {
+    const status_3: string;
+    export { status_3 as status };
+    const code_3: string;
+    export { code_3 as code };
+}
+export namespace CONCERTO_SEMANTIC_EXCEPTION {
+    const status_4: string;
+    export { status_4 as status };
+    const code_4: string;
+    export { code_4 as code };
+}
+export namespace SECURITY_EXCEPTION {
+    const status_5: string;
+    export { status_5 as status };
+    const code_5: string;
+    export { code_5 as code };
+}
+export namespace METAMODEL_EXCEPTION {
+    const status_6: string;
+    export { status_6 as status };
+    const code_6: string;
+    export { code_6 as code };
+}
+export namespace ILLEGAL_MODEL_EXCEPTION {
+    const status_7: string;
+    export { status_7 as status };
+    const code_7: string;
+    export { code_7 as code };
+}
```

Comment:
This is a breaking change, unfortunately. Is there a way to make this backwards compatible? If not, we should target the v4.0 branch

Reply:
Alright, will rebase the PR to v4.0